platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerNodeConfiguration.java

```diff
@@ -31,32 +32,44 @@ public class ContainerNodeConfiguration extends AbstractNodeConfiguration {
             new ObjectMapper(new YAMLFactory().disable(Feature.WRITE_DOC_START_MARKER));
 
     /**
-     * Constructor for the {@link ContainerNodeConfiguration} class.
+     * Constructor for the {@link ContainerNodeConfiguration} class
      *
      * @param lifecycleSupplier a supplier that provides the current lifecycle state of the node
      */
     public ContainerNodeConfiguration(@NonNull final Supplier<LifeCycle> lifecycleSupplier) {
-        super(lifecycleSupplier);
-        overriddenProperties.put(
-                EventConfig_.EVENTS_LOG_DIR,
-                Path.of(CONTAINER_APP_WORKING_DIR, EVENT_STREAM_DIRECTORY).toString());
+        this(lifecycleSupplier, new OverrideProperties());
     }
 
     /**
-     * Updates a single property of the configuration to a {@link List} of {@link NetworkEndpoint}.
-     * Can only be invoked when the node is not running.
+     * Constructor for the {@link ContainerNodeConfiguration} class.
+     *
+     * @param lifecycleSupplier a supplier that provides the current lifecycle state of the node
+     * @param overrideProperties override properties to initialize the configuration with
+     */
+    public ContainerNodeConfiguration(
+            @NonNull final Supplier<LifeCycle> lifecycleSupplier,
+            @NonNull final OverrideProperties overrideProperties) {
+        super(lifecycleSupplier, overrideProperties);
+        overriddenProperties.withConfigValue(
```

Comment:
It's a bit unfortunate to have `overrideProperties` and `overriddenProperties` here. I suggest renaming one of the two.